README.md

```diff
@@ -96,37 +96,34 @@ The Asset Publisher SDK can efficiently acquire the location data on a device an
 ```swift
 import AblyAssetTrackingPublisher
 
-// Initialise a Publisher
+// Initialise a Publisher using the mandatory builder methods
 
 publisher = try? PublisherFactory.publishers() // get a Publisher Builder
     .connection(ConnectionConfiguration(apiKey: ABLY_API_KEY, clientId: CLIENT_ID)) // provide Ably configuration with credentials
-    .log(LogConfiguration()) // provide logging configuration
-    .transportationMode(TransportationMode()) // provide mode of transportation for better location enhancements
-    .delegate(self) // provide delegate to handle location updates locally if needed
```

Comment:
Even though this is optional I'd leave the `delegate()` in the example. Unless that's super obvious for iOS developers, then we can remove it (as an Android dev it wouldn't be obvious to me :sweat_smile: )

Reply:
Hmm yeah, I had an internal fight about this as well. I've decided to stick to the "mandatory methods only" and I left a comment below about the protocols with optional methods, but it's probably not enough. I'll re-add those delegates then 👍

Reply:
[d2166b9](https://github.com/ably/ably-asset-tracking-swift/pull/332/commits/d2166b9b343c509726c2488ad2fd23ff62791938)